great_expectations/data_context/types/resource_identifiers.py

```diff
@@ -19,6 +19,8 @@ def expectation_suite_name(self):
         return self._expectation_suite_name
 
     def to_tuple(self):
+        if self._expectation_suite_name is None:
+            return None,
```

Comment:
I might be misunderstanding the comma here: Is the intention is to return `None, None` or `None`? If it's the first, adding a second `None` would be more explicit.

Reply:
The intention is to return a tuple containing a single None (for which I think this is the standard syntax)...essentially, we still do want a tuple, we just don't want to try to run split on None.

I reviewed other instances of DataContextKey to consider the consistency of their treatment, and I think it is worth adding guards consistently since in some cases to_tuple substitutes a value (like "__none__") for `None` to ensure that all fields in the tuple are strings. Since that hadn't been the case before, I thought it makes most sense to make a minimal change to current behavior. 

I think a complete, thoughtful, standardization is a useful separate step.